next-themes/src/script.template.js

```diff
@@ -11,7 +11,7 @@ export const script = (
   const el = document.documentElement
   const systemThemes = ['light', 'dark']
 
-  function updateDOM(theme: string) {
+  function updateDOM(theme) {
```

Comment:
Note: if there is a concern about loosing typesafety here this solution can be improved by either using jsdoc type comments here or adding a custom esbuild loader to the tsup config,